doc/api/deprecations.md

```diff
@@ -717,24 +717,6 @@ Type: Documentation-only
 The [`ecdh.setPublicKey()`][] method is now deprecated as its inclusion in the
 API is not useful.
 
-### DEP0032: `node:domain` module
```

Comment:
We want to keep this entry, it should say  `Type: End-of-Life`, e.g. see https://github.com/nodejs/node/blob/4712d6083d564255d2a3ad8601bcd639f0fc6e33/doc/api/deprecations.md#L44-L64